src/components/confirmCredentialsForm/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useTranslation } from 'react-i18next';
+import { useUpdateMutation } from 'redux/authApiSlice';
+import { Inputs, Props } from 'components/confirmCredentialsForm/types';
+import { ButtonVariant } from 'components/button/types';
+import { useEffect, useState } from 'react';
+import {
+  ErrorMsg,
+  StyledForm,
+  SubTitle,
+  SubmitBtn,
+} from 'components/sharedUI/form/styles';
+import {
+  isErrorWithMessage,
+  isFetchBaseQueryError,
+} from 'helpers/errorHandler';
+import { useConfirmCredentialsSchema } from './confirmCredentialsFormHook';
+
+export const ConfirmCredentialsForm = ({
+  email_value,
+  full_name,
+  id,
+}: Props) => {
+  const { t } = useTranslation();
+  const [isSet, setIsSet] = useState(false);
+  const [update, { isLoading }] = useUpdateMutation();
+  const confirmSchema = useConfirmCredentialsSchema();
+  const {
+    register,
+    handleSubmit,
+    setError,
+    formState: { errors },
+    reset,
+  } = useForm<Inputs>({
+    defaultValues: {
+      email: email_value,
+      full_name: full_name,
+    },
+    resolver: yupResolver(confirmSchema),
+  });
+  const onSubmit: SubmitHandler<Inputs> = async data => {
```

Comment:
please check the order of the functions ad hooks, you have wrong

Reply:
fixed in [5309628](https://github.com/ZenBit-Tech/faq_team_fe/pull/8/commits/53096286bb449eb7af490481ea7d5d267d756c46)